pom.xml

```diff
@@ -58,8 +58,7 @@
 		<jdk.target>1.8</jdk.target>
 
 		<jersey.version>2.30.1</jersey.version>
-		<jackson.version>2.18.3</jackson.version>
```

Comment:
This comment is a bit vague. I am not sure what "old projects" refers to here because there are no recent breaking changes in jackson. 

Is this an outdated comment?

Reply:
@eddumelendez I'd appreciate your input here